pwiz_tools/BiblioSpec/src/DelimitedFileReader.h

```diff
@@ -233,37 +259,73 @@ template <typename STORAGE_TYPE> class DelimitedFileReader {
         std::string line;
         bool parseSuccess = true;
         std::string errorMsg;
-        
+
+        // Boost tokenizer is overkill for some formats
```

Comment:
What is the rationale for this added complexity? Was the tokenizer a performance bottleneck?

Reply:
Yeah, though that may have been in debug build, I don't recall for sure. Hardklor makes crazy big delimited files and this was slowing me down.

Reply:
Boost tokenizer is slow if it's having to allocate new strings. Can you verify that it was a pure debug build that was slow(er) enough to justify this? Because if the `iterator-debugging=off runtime-debugging=off` fast-debug build avoids that problem, I don't think we need the extra complexity.

Reply:
My debug builds are iterator-debugging=off , not sure about runtime-debugging=off. What's the latter do?

It really was painful.

Reply:
Runtime debugging uses the debug CRT which can help for memory-related errors but slows down memory allocation a lot as well. How far is your strtok approach from just being able to handle the tokenizer case as well? I have no particular allegiance to tokenizer so if what you've got is faster then let's use that and only that. I've been looking at alternatives but subbing those in would probably be more work than getting strtok to work for all the cases.

Reply:
The simple strtok-based reader really is simple - no support for quotes etc. That's sufficient for reading Hardklor output, but not for a lot of other stuff, I don't think we can dump tokenizer without reimplementing it ourselves, which of course would be silly.